scania-digital-design-system/sdds-website
# Improvement/pictogram fix

**Describe pull-request**  
Small improvements to pictogram solution

**Solving issue**  

1. text-align left in icon list
2. Making logic to understand when the user selects all icons to direct to aws URL where all pictograms are already zipped

**How to test**  

1. See if pictogram names are left-aligned in list view (aka boxes view)
2. Select all icons and see if button says Download ALL instead of Download selected. Download ALL triggers immediate download of zipped files of 600 and some megabytes.
